components/discord_bot/actions/list-channels/list-channels.mjs

```diff
@@ -0,0 +1,24 @@
+import common from "../common.mjs";
+
+const { discord } = common.props;
+
+export default {
+  ...common,
+  key: "discord_bot-list-channels",
+  name: "List Channels",
+  description: "Return a list of channels. [See the docs here](https://discord.com/developers/docs/resources/guild#get-guild-channels)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    discord,
+    guildId: {
+      ...common.props.guildId,
+    },
+  },
```

Comment:
Since `guildId` and `discord` are already included in common.props, you can remove this `props` section.